config/routes.rb

```diff
@@ -104,6 +104,9 @@
         get '/' => 'admin#index', as: :admin
         resources :announcements, except: [:show], concerns: :paginatable
         resources :users, only: [:index, :update, :destroy], concerns: :paginatable
+        get '/users/invite' => '/instance/user_invitations#new'
+        post '/users' => '/instance/user_invitations#create'
+        get '/users/invitations' => '/instance/user_invitations#index'
```

Comment:
Should try to standardize the routes and declare it as a resource instead of manually specifying each action

https://github.com/coursemology-collab/coursemology2/blob/master/config/routes.rb#L250-L252